packages/dsv/package.json

```diff
@@ -41,6 +41,7 @@
     "del-cli": "^3.0.1",
     "rollup": "^2.23.0"
   },
+  "types": "./types/index.d.ts",
```

Comment:
[Tests](https://github.com/rollup/plugins/blob/master/packages/buble/test/types.ts) should be added for all the new types files, then added to the package.json scripts as [`"test:ts"`](https://github.com/rollup/plugins/blob/master/packages/buble/package.json#L32)